lib/squire/configuration.rb

```diff
@@ -49,7 +49,7 @@ def source(source = nil, options = {})
     def settings(&block)
       @settings ||= setup
 
-      settings = @namespace ? @settings.get_value(@namespace) : @settings
+      settings = defined?(@namespace) ? @settings.get_value(@namespace) : @settings
```

Comment:
I would suggest rather to use `instance_variable_defined?` than `defined?` which is a Ruby keyword and I rather use method, which may be better fit for refactoring later. So something like `instance_variable_defined?(:@namespace)`

Maybe better course of action would be to initialise it to nil on constructor, but I think this is sufficient.